src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -124,9 +126,11 @@ module Log =
 
 module private Write =
 
-    let private writeEventsAsync (writer: MessageDbWriter) (streamName : string) (version : int64) (events : IEventData<EventBody> array)
-        : Async<MdbSyncResult> =
-        writer.WriteMessages(streamName, events, version, Async.DefaultCancellationToken) |> Async.AwaitTaskCorrect
+    let private writeEventsAsync (writer: MessageDbWriter) (streamName : string) (version : int64 option) (events : IEventData<EventBody> array)
```

Comment:
make this be `task{` like the equivalent level in the ReadLastEvent chain (triggered by Async in name of an Async fn - I tend to only ever use that for Task stuff these days)

Reply:
Not sure this function even needs to exist 🤔 It's just calling the method on the writer

Reply:
I might do a cleanup later, will change this to task for now

Reply:
yes this layer is only really sensible if there is no Client construxt